rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
+
```

Comment:
- [ ] To correctly match the associations, kindly ensure that `Getters` and `Setters` for `@book` and `@person` instance variables are present as well. This for project requirements i.e you should update your attributes to this:
`attr_accessor :date, :book, person`